detekt-psi-utils/src/main/kotlin/dev/detekt/psi/AnnotationExcluder.kt

```diff
@@ -25,37 +25,37 @@ class AnnotationExcluder(
      * which basically describes entries to exclude.
      */
     fun shouldExclude(annotations: List<KtAnnotationEntry>): Boolean =
-        annotations.any { annotation -> annotation.typeReference?.let { isExcluded(it, context) } ?: false }
+        annotations.any { annotation -> annotation.typeReference?.let { isExcluded(it) } ?: false }
 
-    private fun isExcluded(annotation: KtTypeReference, context: BindingContext): Boolean {
-        val fqName = if (context == BindingContext.EMPTY) null else annotation.fqNameOrNull(context)
+    private fun isExcluded(annotation: KtTypeReference): Boolean {
+        val fqName = annotation.fqNameOrNull()
         val possibleNames = if (fqName == null) {
             fullQualifiedNameGuesser.getFullQualifiedName(annotation.text.toString())
                 .map { it.getPackage() to it }
+                .flatMap { (packaage, fqName) ->
```

Comment:
```suggestion
                .flatMap { (package, fqName) ->
```
It's a typo copied from the old class - no point fixing it there since `AnnotationExcluderBindingContext` will be deleted.